src/components/HomeViewComponents/SectionHero.vue

```diff
@@ -50,6 +51,24 @@ import 'bootstrap/dist/js/bootstrap.bundle';
 }
 
 .col-lg-10 {
-  font-size: 1.2rem !important; 
+  font-size: 1.2rem !important;
+  
+}
+.col-md-10{
```

Comment:
you should try to be consistent with the structure. 
You have `.name {` in one case, in second `.name{`
I think you should add space before `{`